ztreamy/server.py

```diff
@@ -1329,19 +1363,37 @@ def stop_server():
                            help='certfile for HTTPS connections')
     tornado.options.define('keyfile', default=None,
                            help='keyfile for HTTPS connections')
+    tornado.options.define('whitelist_sub', default=None,
+                           help='whitelist of IP to subscribe')
+    tornado.options.define('whitelist_pub', default=None,
+                           help='whitelist of IP to publish')
+
     tornado.options.parse_command_line()
     port = tornado.options.options.port
     certfile = tornado.options.options.certfile
     keyfile = tornado.options.options.keyfile
+    whitelist_sub = tornado.options.options.whitelist_sub
+    whitelist_pub = tornado.options.options.whitelist_pub
     if (tornado.options.options.buffer is not None
         and tornado.options.options.buffer > 0):
         buffering_time = tornado.options.options.buffer * 1000
     else:
         buffering_time = None
     server = StreamServer(port, certfile, keyfile,
                  stop_when_source_finishes=tornado.options.options.autostop)
-    stream = Stream('/events', allow_publish=True,
-                    buffering_time=buffering_time)
+    if whitelist_sub is not None:
+        whitelist1 = authorization.IPAuthorizationManager()
+        whitelist1.load_from_file(whitelist_sub)
+    else:
+        whitelist1 = None
+    if whitelist_pub is not None:
+        whitelist2 = authorization.IPAuthorizationManager()
+        whitelist2.load_from_file(whitelist_pub)
+    else:
+        whitelist2 = None
+    stream = Stream('/events', auth_manager_sub = whitelist1, \
```

Comment:
The "\" at the end is unnecessary. Keyword arguments should have no whitespace (`auth_manager_sub=whitelist1`) and there should be a space after each comma.